etcdserver/api/v3rpc/rpctypes/error.go

```diff
@@ -43,6 +46,30 @@ var (
 	ErrRoleNotFound     = grpc.Errorf(codes.FailedPrecondition, "etcdserver: role name not found")
 	ErrAuthFailed       = grpc.Errorf(codes.InvalidArgument, "etcdserver: authentication failed, invalid user ID or password")
 
+	// client-side error
+	ErrGRPCEmptyKey     = Error(ErrEmptyKey)
```

Comment:
this should be ErrTooManyOps. And the current ErrEmptyKey should be ErrGRPCTooManyOps.